engines/gob/degob_script_v1.cpp

```diff
@@ -926,13 +926,72 @@ void Script_v1::o1_palLoad(FuncParams &params) {
 		break;
 
 	case 53:
+		skip(2);
+		return;
+
+	case 55:
+		skip(2);
+		break;
+
+	case 56:
 		skip(2);
 		break;
 
 	case 61:
 		skip(4);
 		break;
 	}
+
+	if ((cmd & 0x7F) == 0x30) {
+		skip(48);
+		return;
+	}
+
+	cmd &= 0x7F;
+
+	if (cmd == 49) {
+		bool allZero = true;
+
+		for (int i = 2; i < 18; i++) {
+			if (peekUint8() != 0) {
+				allZero = false;
+				break;
+			}
+		}
+		if (!allZero) {
+			skip(48);
+			return;
+		}
+
+		for (int i = 0; i < 18; i++) {
+			skip(1);
+		}
+
+		return;
+	}
+
+	switch (cmd) {
+	case 50:
+		for (int i = 0; i < 16; i++)
+			readUint8();
+		break;
+
+		case 52:
+			for (int i = 0; i < 16; i++) {
+				readUint8();
+				readUint8();
+				readUint8();
+			}
```

Comment:
48 bytes were already read above, this seems to break INTRO.TOT in Gobliiins 3

Reply:
Indeed ! The o1_palLoad was not only broken, but most of the added code was not needed at all (I tried to mimic some logic of the o1_palLoad from the main scummvm repo without seeing for degob it gets simplified to just a matter of skipping the right amount of bytes depending on the cmd value). I could reproduced the regression it also on the GOB2 demo, it is fixed now.